docs/source/apps/supermarq/qcvv/qcvv_css.ipynb

```diff
@@ -198,12 +197,12 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Estimated gate error: 0.009902908593895088\n"
+      "Estimated gate error: 0.00992845656304131\n"
```

Comment:
(same comment as [before](https://github.com/Infleqtion/client-superstaq/pull/1109#discussion_r1889056612) re: passing a seed to the simulator)